user_config.c

```diff
@@ -0,0 +1,73 @@
+#include "user_config.h"
+#include <stdio.h>
+
+static uint8_t set_debug_level = 0, change_function = 0, temp_num = 0;
+static bool set_sensor_messages = false;
+static char save_type;
```

Comment:
What does this variable do? Can you leave a quick comment?

Reply:
save_type is the variable used to save what kind of message is being sent (either 'G' or 'S'). I will add a comment explaining this!